packages/marko-web-identity-x/browser/profile.vue

```diff
@@ -317,17 +299,96 @@ export default {
       const { customSelectFieldAnswers } = this.user;
       return isArray(customSelectFieldAnswers) ? customSelectFieldAnswers : [];
     },
+
+    showAddressBlock() {
+      // Don't show at all until country is selected.
+      if (!this.countryCode) return false;
+
+      // Only show if a subfield is visible
+      if (this.citySettings.visible) return true;
+      if (this.streetSettings.visible) return true;
+      if (this.regionCodeSettings.visible) return true;
+      if (this.postalCodeSettings.visible) return true;
+      return false;
+    },
+
+    /**
+     * Field settings
+     */
+    givenNameSettings() {
+      return {
+        required: this.isFieldRequired('givenName'),
+        visible: this.isFieldVisible('givenName'),
+      };
+    },
+    familyNameSettings() {
+      return {
+        required: this.isFieldRequired('familyName'),
+        visible: this.isFieldVisible('familyName'),
+      };
+    },
+    organizationSettings() {
+      return {
+        required: this.isFieldRequired('organization'),
+        visible: this.isFieldVisible('organization'),
+      };
+    },
+    organizationTitleSettings() {
+      return {
+        required: this.isFieldRequired('organizationTitle'),
```

Comment:
Unfortunately can't use the function calls here either otherwise changes to the array won't trigger a change to the computed prop.